test/e2e/Tests/Fixtures/FunctionAppFixture.cs

```diff
@@ -12,166 +9,47 @@ namespace Microsoft.Azure.Durable.Tests.DotnetIsolatedE2E;
 
 public class FunctionAppFixture : IAsyncLifetime
 {
-    private readonly ILogger _logger;
-    private bool _disposed;
-    private Process? _funcProcess;
-    private string? _appName;
+    internal readonly ILogger _logger;
+    internal TestLoggerProvider TestLogs { get; private set; }
 
-    private JobObjectRegistry? _jobObjectRegistry;
+    internal FunctionAppProcess? _functionAppProcess;
 
     public FunctionAppFixture(IMessageSink messageSink)
     {
-        // initialize logging            
         ILoggerFactory loggerFactory = new LoggerFactory();
         this.TestLogs = new TestLoggerProvider(messageSink);
         loggerFactory.AddProvider(this.TestLogs);
-        this._logger = loggerFactory.CreateLogger<FunctionAppFixture>();
-        this._appName = Environment.GetEnvironmentVariable("TEST_APP_NAME") ?? "BasicDotNetIsolated";
+        this._logger = loggerFactory.CreateLogger<FunctionAppProcess>();
     }
 
-    public async Task InitializeAsync()
+    public Task InitializeAsync()
     {
-        // start host via CLI if testing locally
-        if (Constants.FunctionsHostUrl.Contains("localhost"))
+        string? e2eTestLanguageEnvVarValue = Environment.GetEnvironmentVariable("E2E_TEST_FUNCTIONS_LANGUAGE");
+        _logger.LogInformation("E2E_TEST_FUNCTIONS_LANGUAGE set to " + e2eTestLanguageEnvVarValue);
+        switch ((e2eTestLanguageEnvVarValue ?? "").ToLowerInvariant())
```

Comment:
Instead of this approach, where the fixture uses the env var to select which language to launch and where, it may be possible to do a "chimera" approach where we launch many copies of func all at once on different ports, one for each app. Then the tests can run against all of the apps on each run... needs more investigation